assets/js/modules/analytics/common/account-select.test.js

```diff
@@ -0,0 +1,127 @@
+/**
+ * Account Select component tests.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import apiFetchMock from '@wordpress/api-fetch';
+
+/**
+ * Internal dependencies
+ */
+import AccountSelect from './account-select';
+import { fireEvent, render } from '../../../../../tests/js/test-utils';
+import { STORE_NAME as modulesAnalyticsStoreName, ACCOUNT_CREATE } from '../datastore/constants';
+import * as fixtures from '../datastore/__fixtures__';
+
+// Mock apiFetch so we know if it's called.
+jest.mock( '@wordpress/api-fetch' );
+apiFetchMock.mockImplementation( ( ...args ) => {
+	// eslint-disable-next-line no-console
+	console.warn( 'apiFetch', ...args );
+} );
+
+const setupRegistry = ( registry ) => {
+	registry.dispatch( modulesAnalyticsStoreName ).setSettings( {} );
+	registry.dispatch( modulesAnalyticsStoreName ).receiveAccounts( fixtures.accountsPropertiesProfiles.accounts );
```

Comment:
They haven't been added yet although I kept what I started. I stopped working on them when we were on a tighter timeline, but I agree that fixtures have been a bit of a pain to use in component tests. Ideally we would only be using them for Storybook and VRT. If we have time, it would be great to finish those and refactor these.

Reply:
Let's defer that to a later stage. The current way is good enough for now I'd say.